src/components/ListCountries/ListCountries.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable react/no-array-index-key */
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import CountUp from 'react-countup';
+import { NavLink } from 'react-router-dom';
+import { fetchCountries } from '../../redux/worldCountries/worldCountries';
+import styles from './ListCountries.module.css';
+
+const ListCountries = () => {
```

Comment:
- [ ] Your live demo is missing the search functionality you have implemented. Kindly ensure this is rectified.

  ![ismail-search-live](https://user-images.githubusercontent.com/36542986/158455486-2d1f4d6c-9b6a-4f78-82f0-7ceb9a2078c3.jpg)